demos/asp-net-core/ITOps.ViewModelComposition/ISubscription.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition
+{
+    interface ISubscription
```

Comment:
you wanted it 😄 , it was an abstract class before. In any case a non generic type is required here https://github.com/Particular/Workshop.Microservices/blob/asp-net-core-api-gateway/demos/asp-net-core/ITOps.ViewModelComposition/DynamicViewModel.cs#L14 where we need a list of non generic types and generic open types cannot be used (unfortunately)